client/src/components/MenuIcon.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+const MenuBar = styled.div`
+  width: 40px;
+  height: 3px;
+  background-color: ${(props) => props.theme.colors.altTwo};
+  transition: 0.5s;
```

Comment:
```suggestion
  transition: 0.2s;
```
I think faster feels a bit more responsive, just a personal preference